examples/src/test/java/io/grpc/examples/routeguide/RouteGuideServerTest.java

```diff
@@ -86,7 +86,11 @@ public void setUp() throws Exception {
 
   @After
   public void tearDown() {
-    server.stop();
+    try {
+		  server.stop();
```

Comment:
Indentation needs to be fixed.

Reply:
How did those tabs get there...fixed! https://github.com/grpc/grpc-java/pull/6512/commits/91391d2017a0481fbf29f7ecebf510d1e4abfe04